tests/metrics/README.md

```diff
@@ -4,25 +4,19 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo make metrics-tests
```

Comment:
Sadly, it is slightly more complicated - at least one of the tests (docker_memory_usage.sh) requires to be running as `root` to be able to run `smem` whilst looking at the docker processes it does not own - so, removing the `sudo` I expect will make that test not generate the correct data. Maybe we just make a note saying if the build may fail if `root` does not have a correct environment set up, so you may need to first run as non-root to get a valid build, and then again as root to get a full set of valid results. Not great. @jodh-intel , any thoughts on a better solution - non are immediately springing to my mind.
 And, of course (such as in my case), we don't actually do an `install` step here, so we are actually testing the system installed CC components, not the ones we just built afaik :-)  Maybe the make `metrics-tests' rule should not rely on the CC binaries being built, as it is not actually testing those binaries??